lib/_http_outgoing.js

```diff
@@ -337,6 +337,14 @@ OutgoingMessage.prototype.getHeader = function(name) {
 };
 
 
+OutgoingMessage.prototype.getAllHeaders = function() {
+  if (!this._headers)
+    return;
```

Comment:
Good question. I think it was just copied from `getHeader()`.

I had taken it because I thought it was done so that the api was closer to `getHeader()`. I think an object makes more sense.